test/Concurrency/Runtime/async_stream.swift

```diff
@@ -7,9 +7,6 @@
 // rdar://78109470
 // UNSUPPORTED: back_deployment_runtime
 
-// Race condition
-// REQUIRES: rdar78033828
-
```

Comment:
Reenabling the tests since the underlying problem with task cancellation was fixed by @ktoso

Reply:
At least I certainly hope so! This was quite some time ago, thank you for enabling the tests again 💯